support/config/precached.conf

```diff
@@ -189,9 +191,9 @@ program_blacklist = [
   "/usr/bin/wc",
   "/usr/bin/wget",
   "/usr/bin/which",
+  "/usr/bin/xdg-dbus-proxy",
```

Comment:
What are the benefits of blacklisting this?

Reply:
As far as I understand it is used by flatpak and gnome too. In this case proxy from within the sandbox of flatpak to the gnome desktop.

But every flatpak uses it. So you can not profile the proxy itself because you may would generate not so useful rules.

Their are more binaries. Basically everything starting with xdg-* may be worth blacklisting. At least profiles starting with these binaries does not make sense. I think it is better to profile before the proxy/translator and after(inside the sandbox) but not the connection itself. Better cut the prefetch rules at this point.

Reply:
Some rule applies to wine. I see little benefit but a performance penalty in profiling translators

Reply:
```
[idnovic@id-lappy ~]0$ /usr/bin/xdg
xdg-dbus-proxy            xdg-icon-resource         xdg-settings
xdg-desktop-icon          xdg-mime                  xdg-user-dir
xdg-desktop-menu          xdg-open                  xdg-user-dirs-gtk-update
xdg-email                 xdg-screensaver           xdg-user-dirs-update
[idnovic@id-lappy ~]0$ /usr/libexec/gsd
gsd-a11y-settings        gsd-keyboard             gsd-screensaver-proxy
gsd-backlight-helper     gsd-locate-pointer       gsd-sharing
gsd-clipboard            gsd-media-keys           gsd-smartcard
gsd-color                gsd-mouse                gsd-sound
gsd-datetime             gsd-power                gsd-wacom
gsd-disk-utility-notify  gsd-printer              gsd-wacom-led-helper
gsd-dummy                gsd-print-notifications  gsd-wacom-oled-helper
gsd-housekeeping         gsd-rfkill               gsd-xsettings
[idnovic@id-lappy ~]0$ /usr/libexec/gvfs
gvfs-afc-volume-monitor      gvfsd-http
gvfsd                        gvfsd-localtest
gvfsd-admin                  gvfsd-metadata
gvfsd-afc                    gvfsd-mtp
gvfsd-afp                    gvfsd-network
gvfsd-afp-browse             gvfsd-nfs
gvfsd-archive                gvfsd-recent
gvfsd-burn                   gvfsd-sftp
gvfsd-cdda                   gvfsd-smb
gvfsd-computer               gvfsd-smb-browse
gvfsd-dav                    gvfsd-trash
gvfsd-dnssd                  gvfs-goa-volume-monitor
gvfsd-ftp                    gvfs-gphoto2-volume-monitor
gvfsd-fuse                   gvfs-mtp-volume-monitor
gvfsd-google                 gvfs-udisks2-volume-monitor
gvfsd-gphoto2   
```